dummyserver/testcase.py

```diff
@@ -76,6 +76,23 @@ def tearDownClass(cls):
         if hasattr(cls, 'server_thread'):
             cls.server_thread.join(0.1)
 
+    def assert_header_received(
+            self,
+            received_headers,
+            header_name,
+            expected_value=None
+    ):
```

Comment:
I'm not wild about this indentation. :wink: Mind matching it up with the rest of the project?

Reply:
Sure! Another project I'm involved with hand linting set up to whine if the parameters line up with the first line of the function, and that's why I did it here, but I agree it looks skeevy.